src/main/java/com/dev/cinema/dao/CinemaHallDaoImpl.java

```diff
@@ -5,13 +5,17 @@
 import com.dev.cinema.model.CinemaHall;
 import com.dev.cinema.util.HibernateUtil;
 import java.util.List;
+import org.apache.log4j.Logger;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private static final Logger logger = Logger.getLogger(CinemaHallDaoImpl.class);
+
     @Override
     public CinemaHall add(CinemaHall cinemaHall) {
+        logger.info("Trying insert CinemaHall in to database");
         Transaction transaction = null;
         Session session = null;
         try {
```

Comment:
**logger.error()** ?
In every **catch** block

Reply:
exception will be logged anyway